flink-kubernetes/src/main/java/org/apache/flink/kubernetes/cli/KubernetesCliOptions.java

```diff
@@ -91,22 +90,6 @@ public static Option getOptionWithPrefix(Option option, String shortPrefix, Stri
 		.desc("Help for Kubernetes session CLI.")
 		.build();
 
```

Comment:
Removing these two means that the `KubernetesConfigOptionsInternal.ENTRY_POINT_CLASS_ARGS` is always null so we can remove it, right?

Reply:
I opened this https://issues.apache.org/jira/browse/FLINK-15195, so if you agree let me know.

Reply:
Yes, you are right. `KubernetesConfigOptionsInternal.ENTRY_POINT_CLASS_ARGS` is an internal config option. It is used for per-job and yaml submission in the future. If you want, it could be removed.